commcare_connect/templates/opportunity/opportunity_list.html

```diff
@@ -8,10 +8,16 @@
     <div class="mt-5 py-3">
       <h1> Opportunities
         <span class="float-end">
+          {% if request.org_membership.is_viewer %}
+          <button class="btn btn-primary" disabled>
+            <span class="bi bi-plus"></span> Add new
+          </button>
+          {% else %}
           <a class="btn btn-primary"
              href="{% url 'opportunity:init' org_slug=request.org.slug %}" title="Add new opportunity"><span
             class="bi bi-plus"></span> Add new
           </a>
+          {% endif %}
```

Comment:
I think this might work be more concise way that works as well
```suggestion
          <a class="btn btn-primary {% if request.org_membership.is_viewer %}disabled{% else %}href="{% url 'opportunity:init' org_slug=request.org.slug %}" title="Add new opportunity"{% endif %}">
            <span class="bi bi-plus"></span> Add new
          </a>
```

Reply:
I tried to add the disabled attribute on the `<a>` tag but the disabled is not supported on links. The disabled attribute is supported by a button and mostly input related elements. : )